src/app/assets/languages/en.json

```diff
@@ -271,9 +271,13 @@
     },
     "cache-metrics":{
       "current-number-entries":"Current number of entries",
+      "current-number-entries-tooltip":"Displays the number of entries that are stored in the cache.",
       "min-nodes":"Minimum number of nodes",
+      "min-nodes-tooltip":"Displays the minimum number of nodes for the cache.",
       "cache-size-off-heap":"Size of cache in off-heap memory",
-      "cache-size-heap":"Size of cache in heap memory"
+      "cache-size-heap":"Size of cache in heap memory",
+      "cache-size-heap-tooltip":"Displays the amount of JVM heap memory that data in the cache uses. Total memory usage is higher because it includes internal data structures.",
```

Comment:
@danberindei How do we measure "amount"? Is this in Gb or Mb or does it depend? I think that's something we need to clarify because until it hits that threshold the console will display 0, even if there are entries in the cache.